scripts/release/package_audit_lib.rb

```diff
@@ -422,15 +422,18 @@ def check_tool_base(path, base_pkgs)
         # Handle nuances in individual packages
         # Search here to get the URLs: https://cdnjs.com/
         case package
+        when 'vue'
+          `curl https://cdnjs.cloudflare.com/ajax/libs/#{package}/#{latest}/#{package}.global.min.js --output public/js/#{package}.global-#{latest}.min.js`
+          `curl https://cdnjs.cloudflare.com/ajax/libs/#{package}/#{latest}/#{package}.global.prod.min.js --output public/js/#{package}.global.prod-#{latest}.min.js`
         when 'single-spa'
           `curl https://cdnjs.cloudflare.com/ajax/libs/#{package}/#{latest}/system/#{package}.min.js --output public/js/#{package}-#{latest}.min.js`
           `curl https://cdnjs.cloudflare.com/ajax/libs/#{package}/#{latest}/system/#{package}.min.js.map --output public/js/#{package}-#{latest}.min.js.map`
         when 'systemjs'
           `curl https://cdnjs.cloudflare.com/ajax/libs/#{package}/#{latest}/system.min.js --output public/js/#{package}-#{latest}.min.js`
         when 'vuetify'
           FileUtils.rm(Dir["public/css/vuetify-*"][0]) # Delete the existing vuetify css
-          `curl https://cdnjs.cloudflare.com/ajax/libs/#{package}/#{latest}/#{package}.min.css --output public/css/#{package}-#{latest}.min.css`
-          `curl https://cdnjs.cloudflare.com/ajax/libs/#{package}/#{latest}/#{package}.min.js --output public/js/#{package}-#{latest}.min.js`
+          `curl https://cdnjs.cloudflare.com/ajax/libs/#{package}/#{latest}/#{package}-labs.min.css --output public/css/#{package}-labs-#{latest}.min.css`
+          `curl https://cdnjs.cloudflare.com/ajax/libs/#{package}/#{latest}/#{package}-labs.min.js --output public/js/#{package}-labs-#{latest}.min.js`
```

Comment:
I don't currently have the .min version in index.html.  Need to update that if wwe change to the min js version.
Same for vue.